### PR TITLE
feat(cli): add CLI version in addition to runtime versions

### PR DESCRIPTION
**Pull request by nandor-magyar**
CLI version was missing from the output, minor improvement.

### CODE REVIEW

File: .github/workflows/full.yml [hunk 1]
```diff
@@ -576,26 +584,54 @@ jobs:
...
+          make GOOS="linux" GOARCHS="amd64 arm64" compile-agents
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Setup binfmt
+        run: make binfmt
+      - name: Build images
+        run: |
+          ls ./build/out
```
Comment (polaroi8d) [inline .github/workflows/full.yml:611]: ls?

File: .github/workflows/full.yml [hunk 2]
```diff
@@ -608,6 +644,39 @@ jobs:
...
+          path: |
+            $GOPATH/cache
+            $GOPATH/pkg/mod
+          key: ${{ runner.os }}-golang-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-golang-
+      # fixes: fatal: unsafe repository
+      - name: Adding workspace
+        run: git config --global --add safe.directory "$GITHUB_WORKSPACE"
+      - name: Load go mod
```
Comment (polaroi8d) [inline .github/workflows/full.yml:661]: Mixin `go` ang `Golang`, be consistent please.

File: golang/Makefile [hunk 3]
```diff
@@ -137,27 +138,32 @@ lint:
 .PHONY: build-cli
 build-cli:
 	cd build && \
-	docker build -t ${ORG_REGISTRY}/cli/dyo:$(image_version) -t ${DOCKER_REGISTRY}/dyo:$(image_version) -f cli/Dockerfile .
+	docker buildx build  --build-arg REVISION=$$(git rev-parse --short HEAD) --platform=linux/amd64 --load -t ${ORG_REGISTRY}/cli/dyo:$(image_version) -t ${DOCKER_REGISTRY}/dyo:$(image_version) -f cli/Dockerfile .
+
+.PHONY: build-cli-push
```
Comment (polaroi8d) [inline golang/Makefile:143]: Don't we want to add a comment for this?

**Review by polaroi8d — CHANGES_REQUESTED** [review]

File: .github/workflows/full.yml (hunk 1 above)
Comment (nandor-magyar) [reply .github/workflows/full.yml]: Removed the change.

File: .github/workflows/full.yml (hunk 2 above)
Comment (nandor-magyar) [reply .github/workflows/full.yml]: Golang is the language, go.mod is the lock file, but could be more verbose.

Comment (m8vago) [reply .github/workflows/full.yml]: I think it's fine

**Review by polaroi8d — APPROVED** [review]

**Pull request merged**